task03/task03/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task03
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите трёхзначное число: ");
+
+            int i = Convert.ToInt32(Console.ReadLine());
+
+            int a = i / 100;
+            int b = ((i - (i / 100) * 100)) / 10;
```

Comment:
Не проще ли так:
```suggestion
            int b = (i / 10) % 10;
```
?